sqlFile.js

```diff
@@ -0,0 +1,43 @@
+var sqlite3 = require("sqlite3").verbose();
+var db = new sqlite3.Database("url-shortner.db");
```

Comment:
nit: ideally you would use an environment variable to configure the file name for the database file.